formats/application/vnd/oada/isoblue/can/1/+json/example/default.js

```diff
@@ -0,0 +1,55 @@
+module.exports = {
+    _type: 'application/vnd.oada.isoblue.can.1+json',
+  
+    indexing: [
+      {
+        name: 'year-index',
+        source: 'oada.vocab.year-index',
+        value: '2019',
+      },
+      { 
+        name: 'day-index',
+        source: 'oada.vocab.day-index',
+        value: '2019-05-04',
+      },
+      {
+        name: 'hour-index',
+        source: 'oada.vocab.hour-index',
+        value: '22:00',
+      },
+    ],
+ 
+    templates: {
```

Comment:
You don't have anything template-able in this example, and the example you've given of a template is just a data point.  Templates are supposed to be things that are repeated among multiple data samples, therefore you simplify by moving the common things out to a template.  In this case, I'd just remove the template from your example.

Reply:
Changed template to `time` unit (`unix-timestamp`) only